packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/components/EmptyTreeContent.tsx

```diff
@@ -13,9 +13,22 @@ interface EmptyTreeContentProps {
 /** @internal */
 export function EmptyTreeContent({ icon }: EmptyTreeContentProps) {
```

Comment:
Maybe this should be moved to `Warnings` file and renamed to `WarningEmptyTree`? It also could reuse `tw-filter-warning-container` css class.

Reply:
We can move it, but it should not reuse tw-filter-warning-container as the warning for empty should be centered.

Reply:
So you could add a separate class that add aditional styling to center it. For me it looks like they are sharing majority of css.